Testing/UnitTests/xASL_ut_function_xASL_bids_CheckDatasetDescription.m

```diff
@@ -0,0 +1,100 @@
+function UnitTest = xASL_ut_function_xASL_bids_CheckDatasetDescription(TestRepository)
+%xASL_ut_function_xASL_bids_CheckDatasetDescription Individual unit test for xASL_bids_CheckDatasetDescription
+%
+% INPUT:        TestRepository - Path to test repository.
+%
+% OUTPUT:       UnitTest  - Test structure
+%               name      - Name of tested module or submodule (char array)
+%               unit      - Insert one of the following: 'Module', 'Submodule' or 'Function'
+%               passed    - Result of all subtests combined (true or false)
+%               test      - Structure with individual subtest results
+%
+% -----------------------------------------------------------------------------------------------------------------------------------------------------
+% DESCRIPTION:  Should be run using xASL_ut_UnitTesting.
+%
+% EXAMPLE:      UnitTests(1) = xASL_ut_function_xASL_bids_CheckDatasetDescription(TestRepository);
+% -----------------------------------------------------------------------------------------------------------------------------------------------------
+% Copyright 2015-2021 ExploreASL
+
+
+%% Test run 1
+
+% Give your individual subtest a name
+UnitTest.tests(1).testname = 'Correct version, different version, non-xASL import, missing field';
+
+% Start the test
+testTime = tic;
+
+% Run your test here
+datasetDescription.Name = 'TestPatient';
+datasetDescription.BIDSVersion = '1.6.0';
+datasetDescription.DatasetType = 'raw';
+datasetDescription.HowToAcknowledge = 'Please cite this paper: https://www.ncbi.nlm.nih.gov/pubmed/001012092119281';
+
+% Example 1: correct version
+datasetDescription.Acknowledgements = 'Imported with ExploreASL 1.8.0_BETA';
+xVersion = '1.8.0_BETA';
+[bImportedExploreASL1, bImportedSameVersion1, versionExploreASLBIDS1, bImportedBETA1] = xASL_bids_CheckDatasetDescription(datasetDescription,xVersion);
+
+% Example 2: different version
+datasetDescription.Acknowledgements = 'Imported with ExploreASL 1.7.0';
```

Comment:
Add fullstop, and random text after it...

Reply:
And probably also before.

Reply:
I added an extra test case 773a8c2 👍